source/DemoScene.cpp

```diff
@@ -80,5 +85,16 @@ bool DemoScene::update(Game &)
   if (Keyboard::getKeys()[OIS::KC_S]) {
     cameraNode->translate(0, 0, 10);
   }
+
+  if (Keyboard::getKeys()[OIS::KC_U]) {
+    illidan.stopAnimation();
+  }
+  if (Keyboard::getKeys()[OIS::KC_O]) {
+    illidan.setAnimation("Move", false);
+  }
+  if (Keyboard::getKeys()[OIS::KC_P]) {
+    illidan.setAnimation("Stand", false);
+  }
+  illidan.updateAnimations(fe.timeSinceLastFrame);
```

Comment:
Is skipping the beginning of the animation not a problem?
When you change animation the timer is reset (if I understood right), so the difference is do we start at 0 or at 0 + time of the first frame.